src/components/ProjectCard/index.jsx

```diff
@@ -0,0 +1,78 @@
+import {
+  withStyles,
+  Card,
+  CardContent,
+  CardActions,
+  Typography,
+} from 'material-ui';
+import { Component } from 'react';
+import { object, node } from 'prop-types';
+import ReactMarkdown from 'react-markdown';
+
+@withStyles(
+  theme => ({
+    card: {
+      marginBottom: 30,
```

Comment:
Since we added the `spacing` to the Grid container, we don't need to have `marginBottom` anymore.